tests/rados/test_track_availibility_status.py

```diff
@@ -0,0 +1,604 @@
+import datetime
+import math
+import random
+import time
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from tests.rados.monitor_configurations import MonConfigMethods
+from tests.rados.stretch_cluster import wait_for_clean_pg_sets
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw) -> int:
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_object = RadosOrchestrator(node=cephadm)
+    mon_obj = MonConfigMethods(rados_obj=rados_object)
+    pool_obj = PoolFunctions(node=cephadm)
+    client = ceph_cluster.get_nodes(role="client")[0]
+
+    replicated_config = config.get("replicated_pool")
+    pool_name = replicated_config["pool_name"]
+
+    try:
+        assert mon_obj.set_config(
+            section="global", name="osd_pool_default_pg_autoscale_mode", value="off"
+        ), "Could not set pg_autoscale mode to OFF"
+
+        if not rados_object.create_pool(**replicated_config):
+            log.error("Failed to create the replicated Pool")
+            return 1
+
+        rados_object.bench_write(pool_name=pool_name, byte_size="5K", max_objs=10000)
+        msg_data_push = f"The data pushed into the {pool_name} pool"
+        log.info(msg_data_push)
+        log.info(
+            "==== Case 1: Verification of the Mean Time Between Failures (MTBF) value ===="
+        )
+
+        result_MTBF = test_MTBF_value(client, pool_name)
+        if not result_MTBF:
+            log.error("Case 1: Verification of the MTBF failed ")
+
+        log.info(
+            "==== Case 1: Verification of the Mean Time Between Failures (MTBF) value is completed ===="
+        )
+
+        log.info(
+            "==== Case 2: Verification of the Mean Time To Recover(MTTR) value ===="
+        )
+
+        result_MTTR = test_MTTR_value(client, pool_name)
+
+        if not result_MTTR:
+            log.error("Case 2: Verification of the MTTR failed ")
```

Comment:
please add return / exit statement here, in case of failures.

Reply:
Done.